### PR TITLE
User profile page

### LINKED ISSUE #3: User profile page
Path: `/user/:login/`

This page should include: github avatar, link to github acc, link to twitter acc, link to announcements created by this user.

Also, there should be another link in nav `/me` for signed-in users which directs user to his user profile page.

### PR DESCRIPTION
**Pull request by veelenga**
Fixes #3, #5 

Profile page url: `/users/:login`
Search by user login url: `/?user=login&type=blog_post`

* [x] user profile page
* [x] filter announcements by user
* [x] `/me` path for `current_user`

### CODE REVIEW

**Comment by veelenga** [comment]
Deployed to staging: https://crystal-ann-staging.herokuapp.com/
@hugoabonizio please let me know what do you think about this 😄

File: src/views/layouts/application.slang [hunk 1]
```diff
@@ -24,10 +24,10 @@ html
         div.secondary
           div
             == render_template "layouts/_nav.slang"
-            == render_template "layouts/_social_nav.slang"
-            == render_template "layouts/_search.slang"
             - if signed_in?
               == render_template "layouts/_profile.slang"
+            == render_template "layouts/_search.slang"
+            == render_template "layouts/_social_nav.slang"
```
Comment (hugoabonizio) [inline src/views/layouts/application.slang]: That's exactly what I was going to suggest!

**Review by hugoabonizio — COMMENTED** [review]
Very good! :smile: 

I think your screen height is greater than mine, and maybe the navbar items are too distant in my screen

Before:
image

After:
image

```
-       .main-navigation {
-               margin-bottom: 11.1111%;
+       #site-navigration.main-navigation {
+               margin-bottom: 10%;

-               margin: 0 20% 20%;
+               margin: 0 20% 15%;
```

What do you think? This was a problem only before your last commit when the profile options was at the bottom of the navbar, now it's just a minor detail.

Good job! :satisfied:

**Comment by veelenga** [comment]
Yes, you are right. We need to save space now on a sidebar 😄 BTW, in the nearest future, I was going to add some other "widgets"  below the search field. Maybe something like "Random Announcement" and "Popular tags (types)"... Will discuss this when it comes to business.

I will adjust some margin values. Thanks for having time to take a look 👍

**Comment by hugoabonizio** [comment]
That's a great idea! For measuring popular ones we can use view count or some "star" system like github repos, and each week we can send the top popular as a newsletter by email. Would fill the gap of the crystal-weekly!

We can list some feature ideas later :blush:

**Pull request merged**